src/components/ExportLicenseAsCSVModal/ExportLicenseAsCSVModal.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { mapValues, pickBy, sortBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  Checkbox,
+  Layout,
+  Modal,
+  ModalFooter,
+} from '@folio/stripes/components';
+
+import css from './ExportLicenseAsCSVModal.css';
+
+export default class ExportLicenseAsCSVModal extends React.Component {
+  static propTypes = {
+    onClose: PropTypes.func,
+    onCompareLicenseTerms: PropTypes.func,
+    selectedLicenses: PropTypes.arrayOf(PropTypes.string),
+    terms: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.licenseInformationArray = ['name', 'startDate', 'endDate', 'status', 'type'];
```

Comment:
It seems weird to have `this.fooArray` and `this.foo` for these things. Can we just have a single array that contains objects with `key` and `value` properties? Eg, `this.licenseInformation = [{ key: 'name', label: <FormattedMessage id=ui-licenses.exportLicensesModal.name /> }, ....]`

I'm just thinking of how whenever I do things like this and end up back in the code a year later I'm all "wtf is this one for? what's the one for doing ____ called again?" 😆

Reply:
Would let you avoid string interpolation like [L97](https://github.com/folio-org/ui-licenses/pull/299/files#diff-e89ea18304a649a2ce20769afd4f0f3fR97)

Reply:
Yeah okay.. I guess I tried to be too dynamic here..

Reply:
Yeah I definitely appreciate where you're coming from, I've written stuff like this _so_ many times. And that's really why it scares me, because I've been burned by it like, probably every single time.